browsers.config.js

```diff
@@ -8,14 +8,13 @@ const defaultBrowsers = ['chrome', 'firefox'];
 const availableBrowsers = ['chrome', 'firefox', 'opera'];
 
 // BrowserStack browsers
-const defaultBSBrowsers = ['chrome', 'firefox', 'edge'];
-const previousBSBrowser = ['chrome_previous', 'firefox_previous', 'edge_previous'];
-const supportedBSBrowsers = [...defaultBSBrowsers, ...previousBSBrowser];
+const defaultBSBrowsers = ['chrome', 'firefox', 'safari'];
+const previousBSBrowser = ['chrome_previous', 'firefox_previous', 'safari_previous'];
+const supportedBSBrowsers = [...defaultBSBrowsers, ...previousBSBrowser, 'edge', 'edge_previous'];
```

Comment:
Also, I find this config very confusing. Why `'edge', 'edge_previous'` are in `supportedBSBrowsers` but not `edge` in `defaultBSBrowsers` and `edge_previous` in `previousBSBrowser` ?